ballerina-tests/tests/03_bidirectional_streaming_client.bal

```diff
@@ -251,7 +251,7 @@ isolated function testBidiStreamingWithNoPublicCertFile() returns grpc:Error? {
     test:assertEquals((<grpc:Error>result).message(), "Certificate file location must be provided for secure connection.");
 }
 
-@test:Config {enable:true}
+@test:Config {enable:false}
```

Comment:
Is it related to the recent intermittent test failure in windows?

Reply:
Yes, we need to check this separately.